ingestion/setup.py

```diff
@@ -23,7 +23,8 @@ def get_long_description():
 
 
 base_requirements = {
-    "openmetadata-ingestion-core==0.6.0.dev0",
+    "incremental",
+    "openmetadata-ingestion-core~=0.8.0.dev0",
```

Comment:
@akash-jain-10 `~=` should be compatible with every minor bump like dev0 -> dev1 and so on, right?

Reply:
As per PEP440, it will be compatible. Ref - https://www.python.org/dev/peps/pep-0440/#compatible-release